hack/env.sh

```diff
@@ -16,6 +16,8 @@
 CSV_DIR="config/manifests/bases"
 TEST_CONFIG_FILE="test/.default_config"
 
+DEPENDENT_CRDS_KEYS=(grafana hyperfoil infinispan kafka keycloak knative kogito mongodb prometheus)
```

Comment:
Prometheus CRD shouldn't be deleted as it is used for cluster monitoring.

Reply:
Separated CRDS into sensitive and non-sensitive arrays